lib/resources/example_json/dstu2_examples_contract.rb

```diff
@@ -3,7 +3,7 @@ module Resources
 
     DSTU2_CONTRACT_ENTRY ||= {
       "resourceType": "Contract",
-      "id": "11325025",
+      "id": "7470218",
```

Comment:
Fixed in this [commit](https://github.com/cerner/fhir.cerner.com/pull/94/files#diff-7d5282157e9d95c6569ac37dbc6aba6dR113)